[2024-11-01T06:27:43.972+0000] {processor.py:186} INFO - Started process (PID=5916) to work on /usr/local/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-11-01T06:27:43.974+0000] {processor.py:914} INFO - Processing file /usr/local/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-11-01T06:27:43.976+0000] {logging_mixin.py:190} INFO - [2024-11-01T06:27:43.975+0000] {dagbag.py:588} INFO - Filling up the DagBag from /usr/local/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-11-01T06:27:43.980+0000] {logging_mixin.py:190} INFO - [2024-11-01T06:27:43.979+0000] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-11-01T06:27:43.980+0000] {logging_mixin.py:190} INFO - [2024-11-01T06:27:43.980+0000] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-11-01T06:27:43.981+0000] {processor.py:927} WARNING - No viable dags retrieved from /usr/local/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-11-01T06:27:43.995+0000] {processor.py:208} INFO - Processing /usr/local/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
